wordpress-mobile/WordPress-Android
# [Draft] [GB-Mobile] Activate Preformatted block

This is the WP-Android side PR of https://github.com/wordpress-mobile/gutenberg-mobile/pull/1442 
that adds Preformated block to the list of available blocks in GB-Mobile.

To test:
See testing steps in the [GB-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/1442).

**Note:** Make sure to target the release branch.

cc @marecar3 


PR submission checklist:
